docs/concepts/garbagecollector.md

```diff
@@ -0,0 +1,72 @@
+# Garbage Collector For Immutable `ConfigMap`s/`Secret`s
```

Comment:
Nice doc, well done!

Can you add a sentence about how the behavior of the MR controller changes, when labelling cm/secrets with the GC label?